.github/actions/release/changelog-generator/action.yml

```diff
@@ -43,13 +59,28 @@ runs:
           --project ${{ env.GITHUB_REPOSITORY_PROJECT }} 
           --token ${{ inputs.github-token }} 
           --future-release ${{ inputs.future-release }} 
+          ${{ env.SINCE_ARG }}
           ${{ inputs.optional-arg }}
 
     - name: Populate changelog descriptions
       shell: bash
       run: |
         .github/workflows/scripts/populate-changelog-descriptions.sh ${{ inputs.changelog-path }}
 
+    - name: Cut other releases
+      if: ${{ inputs.output-only-current-release }}
+      shell: bash
+      run: |
+        ESCAPED_CURRENT_VERSION=${${{ inputs.previous-release }}//./\\.}
+        # cut everything before the current release changes
+        sed -n -z -E "s/.*$ESCAPED_CURRENT_VERSION\)(.*)/\1/p" -i ${{ inputs.changelog-config-path }}
+        
+        if [[ ${{ inputs.previous-release }} ]]; then
+          ESCAPED_PREVIOUS_VERSION=${${{ inputs.previous-release }}//./\\.}
+          # cut everything after the current release changes based on the previous version entry
+          sed -n -z -E "s/(.*)(## \[$ESCAPED_PREVIOUS_VERSION.*)/\1/p" -i ${{ inputs.changelog-config-path }}
+        fi
```

Comment:
Wouldn't it be easier to do what the other workflows do:

- https://github.com/iotaledger/identity.rs/blob/dev/.github/workflows/shared-create-dev-release-pr.yml
- https://github.com/iotaledger/identity.rs/blob/dev/.github/workflows/shared-create-main-release-pr.yml
- https://github.com/iotaledger/identity.rs/blob/dev/.github/workflows/shared-create-hotfix-pr.yml

and just extend the excluded tags to exclude every tag other than the latest release?

https://github.com/iotaledger/identity.rs/blob/38c27a99f09977ae703758a0f0a8dc4aa2440c8f/.github/workflows/shared-create-main-release-pr.yml#L70

Reply:
started out with that thought as well 👍 Thing is, it would then group all changes of the excluded versions under the current version. Which does make sense and is something we rely on when we release the changes from many `dev` releases as a proper version and expect the changes form the `dev` release to all be included.